### PR TITLE
Lf 2804 manual irrigation task

### PR DESCRIPTION
**Pull request by wasswarichard**
Add default columns to store boolean values and include them in the post request.
https://lite-farm.atlassian.net/browse/LF-2804?focusedCommentId=14497

### CODE REVIEW

File: packages/api/db/migration/20230106112659_add_extra_columns_irrigation_table.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    const irrigation_type_translation_key = row.irrigation_type_name
+      .toUpperCase()
+      .split(' ')
+      .join('_');
+    await knex('irrigation_type')
+      .update({ irrigation_type_translation_key })
+      .where({ irrigation_type_id: row.irrigation_type_id });
+  }
+
+  await knex.schema.alterTable('location_defaults', (table) => {
```
Comment (Duncan-Brain) [inline packages/api/db/migration/20230106112659_add_extra_columns_irrigation_table.js:26]: @wasswarichard 

The opposite of this should be in 'down' function.

**Comment by Duncan-Brain** [comment]
@wasswarichard 

Before I continue the down function should be fixed so I do not corrupt my database in testing.

Also For your tables, please check and add as many constraints to as many columns as make sense like .notNullable(). This would also be reflected in required sections of Models etc.

File: packages/api/db/migration/20230106112659_add_extra_columns_irrigation_table.js (hunk 1 above)
Comment (wasswarichard) [reply packages/api/db/migration/20230106112659_add_extra_columns_irrigation_table.js:26]: @Duncan-Brain this has been resolved

**Comment by wasswarichard** [comment]
@Duncan-Brain we accept null  irrigation_type_id values for new locations since values have not been set

**Pull request merged**